core/sql/cli/Statement.cpp

```diff
@@ -2102,6 +2102,30 @@ Statement * Statement::getCurrentOfCursorStatement(char * cursorName)
 
 }
 
+//////////////////////////////////////////////////////////////////
+////
+//// A helper function to show buffer in HEX
+////
+//// ///////////////////////////////////////////////////////////////
+
+static char *stringToHex(char * out, Int32 outLen, char * in, Int32 inLen)
```

Comment:
thanks @sureshsubbiah , I was not able to find a common header file. ExpError.h is a good one. I will move this function into there.